easybuild/easyconfigs/r/Rosetta/Rosetta-2015.31.58019-intel-2015a.eb

```diff
@@ -13,8 +13,4 @@ sources = ['%(namelower)s_src_%(version)s_bundle.tgz']
 
 builddependencies = [('SCons', '2.3.6', '-Python-2.7.9')]
 
-modextravars = {
-    'ROSETTA3_DB': '$root/database',
```

Comment:
the correct way would have been to use `%(installdir)s/database`, but it's better to do this in the existing Rosetta easyblock, cfr. https://github.com/hpcugent/easybuild-easyblocks/pull/1092